test/presence.js

```diff
@@ -0,0 +1,93 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe.only('presence', function () {
+  context('given an object', function () {
+    specify('should return the object', function () {
+      const foo = { foo: 'foo' };
```

Comment:
Can we name the parameter `val`?
```suggestion
      const val = { foo: 'foo' };
```